es-ES/contributing.md

```diff
@@ -0,0 +1,43 @@
+# Contribuir a AVA
```

Comment:
As the [French version](https://github.com/sindresorhus/ava-docs/edit/master/fr_FR/contributing.md), add a header for the link to the original and to have a link on the differences, for example markdown formatted (Note, I do not speak Spanish, it is a translation from google translate) :

```markdown
___
**Nota del traductor**

Esta es la traducción del archivo [contributing.md](https://github.com/sindresorhus/ava/blob/master/contributing.md). Aquí hay un [enlace](https://github.com/sindresorhus/ava/compare/65ae07c76b2a5927e6bcc00630a691c98f6e7c46...master#diff-cc4aac3e9be04e0413c9520f223b493c) a las diferencias con el master AVA (Si al hacer clic en el enlace, usted no puede encontrar el archivo `contributing.md` de los archivos modificados, se puede deducir que la traducción está actualizada).
___
```

1. Where the link `contributing.md` points to the file contributing.md from AVA
2. Where the link `enlace` points to the last commit respective translation (`65ae07c76b2a5927e6bcc00630a691c98f6e7c46`) followed by `...master#diff-` and sha file (`cc4aac3e9be04e0413c9520f223b493c`)

That is to do on the 3 files : contributing.md, maintaining.md and readme.md